cdserver/db/seeds.rb

```diff
@@ -1,3 +1,45 @@
 User.create!(email: "sample_user@dhs.nyc.gov", password: "password")
 User.create!(email: "some_person@dhs.nyc.gov", password: "swordfish")
 User.create!(email: "anotherone@dhs.nyc.gov", password: "example")
+
+ew1 = ShelterBuilding.create!(name: 'EW Bldg 1')
+kelly = ShelterBuilding.create!(name: '730 Kelly Street')
```

Comment:
This one does not have a name.  The class should automatically get its name from street_address1
730 Kelly Street is not its name.   Makes sense?